modules/ROOT/pages/package-task-mmp.adoc

```diff
@@ -15,6 +15,8 @@ You can use the Mule Maven Plugin to package your application into a deployable
 
 The packager then packages your application and creates the deployable JAR file into the target directory within your project's folder. +
 
+==== Import your generated JAR to Anypoint Studio.
```

Comment:
"Import your generated JAR to Anypoint Studio." -> "Import Your Generated JAR to Anypoint Studio" 

There is a question of which title style we use, but I think the above is pretty consistent with what we do elsewhere in the docs. For more info, see https://capitalizemytitle.com/ I'll talk to Mysti to help clarify this in the style guide.